### DISCUSSION TITLE
Ask me anything

### DISCUSSION THREAD

**Discussion by IsHereZahin** [IsHereZahin/isherezahin.old#1] [Q&A] [answered]
## 👋 **Welcome**!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * **Ask questions you’re wondering about.**
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ridowan2050** [comment]
Your web design looks great, but the code could be cleaned up a bit more to make it easier to understand.

**Reply by IsHereZahin** [reply]
Thank you for your suggestion, brother. Inshallah, I will fix this.

**Comment by 1chooo** [comment]
Hi, I'm Hugo. Your design looks awesome. May I ask you how to make the design of the cover of project like your style? Which app do you make these figure? Is it canva? If yet could you share the templates? I will really appreciate! Thanks.

<img width="1122" alt="Screenshot 2024-10-28 at 2 57 53 PM" src="https://github.com/user-attachments/assets/1e37fd96-f780-4397-b93b-599974bb8b7a">

**Reply by IsHereZahin** [reply, marked as answer]
Hi **Hugo**,

Thanks for your kind words! I actually created that design long ago, so I can't remember it exactly. For recent mockups, I usually use tools like [Smartmockups ](https://smartmockups.com/)and [Pixelied](https://pixelied.com/)—both work well for high-quality designs. Canva is also a solid choice if you’re comfortable with it. Best of luck with your design, and feel free to reach out if you have more questions!
